Source/ImPlotWidgets/Private/HistogramWidget.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) Coconut Lizard Limited. All rights reserved.
+
+#include "ImPlotWidgets/Public/HistogramWidget.h"
+
+#include <ImGui/Public/ImPlotImplementation.h>
+#include <Runtime/Core/Public/Algo/Sort.h>
+
+void SHistogramWidget::Construct(const FArguments& InArgs)
+{
+	SImPlotWidget::Construct(InArgs);
+}
+
+void SHistogramWidget::OnImGuiDraw() const
+{
+	SImPlotWidget::OnImGuiDraw();
+
+	const FGeometry& AllottedGeometry = ImGuiCustomWidget.Get()->GetTickSpaceGeometry();
+
+	constexpr int32 GraphFlags = ImPlotFlags_AntiAliased;
+	constexpr int32 AxisFlags = ImPlotAxisFlags_NoGridLines;
+
+    // ---------------------------------
+    {
+		static int bins = 50;
+		ImGui::SetNextItemWidth(AllottedGeometry.GetLocalSize().X);
+		if (bins >= 0) 
+		{
+			ImGui::SameLine();
+			ImGui::SetNextItemWidth(AllottedGeometry.GetLocalSize().X);
+			ImGui::SliderInt("##Bins", &bins, 1, 100);
+		}
+
+        float rmin = FMath::Min(SImPlotWidget::InData);;
+        float rmax = FMath::Max(SImPlotWidget::InData);
```

Comment:
Indenting looks wrong here and also need to capitalise variable names. I'm not sure what the 'r' means but I'm guessing we want RMin and RMax?

Reply:
Don't know what's up with the indentations, they look correct for me in visual studio? (Also ImPlot's default variable names, I just didn't realise to rename them)